### PR TITLE
Remove hash link to "Software Info" as lazy loading routes broke it

### PR DESCRIPTION
**Pull request by rhamilto**
...and we plan to migrate the software info details to a modal anyhow.

Fixes https://jira.coreos.com/browse/CONSOLE-788

### CODE REVIEW

**Comment by spadgett** [comment]
We were planning on switching to an about dialog for software version anyway. We might consider just removing the link.

**Comment by rhamilto** [comment]
/hold

Gonna remove the link entirely

File: frontend/public/components/app.jsx [hunk 1]
```diff
@@ -129,8 +130,8 @@ class App extends React.PureComponent {
         <GlobalNotifications />
         <Switch>
           <Route path={['/all-namespaces', '/ns/:ns',]} component={RedirectComponent} />
-          <LazyRoute path="/status/all-namespaces" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <LazyRoute path="/status/ns/:ns" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
+          <Route path="/status/all-namespaces" exact component={ClusterOverviewPage} />
+          <Route path="/status/ns/:ns" exact component={ClusterOverviewPage} />
```
Comment (spadgett) [inline frontend/public/components/app.jsx]: You should be able to revert these changes now

Comment (rhamilto) [reply frontend/public/components/app.jsx]: D'oh.  Thanks.

**Comment by rhamilto** [comment]
Fixed.
